src/components/themedComponents/index.tsx

```diff
@@ -60,6 +60,23 @@ export const SubmitButton = styled(Button)`
   font-size: 16px;
 `;
 
+export const GreenButton = styled(Button)`
```

Comment:
Should this component just be used everywhere? Like for login and on all form submission?

Reply:
Probably - https://app.clickup.com/t/200h9q5